src/models/setting.model.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  BelongsTo,
+  Column,
+  Model,
+  Table,
+  DeletedAt,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+  AllowNull,
+} from 'sequelize-typescript';
+import Account from 'src/models/account.model';
+
+@Table({
+  indexes: [{ fields: ['account_id', 'name'], unique: true }],
+})
+export default class Setting extends Model {
```

Comment:
Id column missing, reason mentioned in some other comment

Reply:
Fixed